docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -507,10 +507,11 @@ JSON-Pointer. For more details, see
 {: .note}
 
 To consume an Extended Resource in a Pod, include the resource name as a key
-in the `spec.containers[].resources.requests` map.
+in both the maps, `spec.containers[].resources.requests` and 
+`spec.containers[].resources.limits`, in the container spec.
```

Comment:
well... it is perfectly acceptable to just specify limits. Kubernetes has builtin logic that defaults `requests` to `limits`.